packages/lit-dev-content/site/docs/templates/expressions.md

```diff
@@ -207,9 +214,11 @@ If the expression makes up the entire attribute value, you can leave off the quo
 html`<img src="/images/${this.image}">`;
 ```
 
-### Setting attributes if data is defined { #ifDefined }
+### Setting attributes if data is available { #attribute-available }
```

Comment:
I think this section might be better re-conceptualized as "Removing an attribute under certain conditions", and I think we should also demote `ifDefined` and instead start by describing that `nothing` removes attributes (this is the key primitive). 

And then maybe add a footnote about `ifDefined` that says "because typically `undefined` and `null` shouldn't ever be rendered as strings, we provide an `ifDefined()` helper that returns `value ?? nothing`, and can be used like this... [example].